manifests/1.2.0/opensearch-dashboards-1.2.0.yml

```diff
@@ -18,28 +18,28 @@ components:
   ref: "main"
```

Comment:
Is `opensearch-dashboards-functional-test` intended to be `main` and not tag?

Reply:
Good question, that repo is in a development stage for testing purposes basically official support for cypress has not been added to OpenSearch Dashboards. So it bundles a testing framework with the build for verification steps but has no impact in the actual deliverables.